### PR TITLE
Change total score for the content assessor

### PR DESCRIPTION
**Pull request by atimmer**
This makes it possible to justify the aggregate score of the content
assessor.

### CODE REVIEW

File: js/contentAssessor.js [hunk 1]
```diff
@@ -45,7 +50,64 @@ var ContentAssessor = function( i18n, options ) {
...
+		return 30;
+	}
+
+	var totalNegativePoints = this.calculateNegativePoints();
+
+	// Determine the total score based on the total negative points.
+	if ( totalNegativePoints < 2 ) {
+
+		 // A green indicator.
+		totalScore = 90;
```
Comment (andizer) [inline js/contentAssessor.js]: Why not return this directly?

File: js/contentAssessor.js [hunk 2]
```diff
@@ -45,7 +50,64 @@ var ContentAssessor = function( i18n, options ) {
...
+
+	// If you have no content, you have a red indicator.
+	if ( results.length === 0 ) {
+		return 30;
+	}
+
+	var totalNegativePoints = this.calculateNegativePoints();
+
+	// Determine the total score based on the total negative points.
+	if ( totalNegativePoints < 2 ) {
```
Comment (andizer) [inline js/contentAssessor.js]: This total if, else statement could be simplified.

File: js/contentAssessor.js [hunk 3]
```diff
@@ -45,7 +50,64 @@ var ContentAssessor = function( i18n, options ) {
...
 };
 
-module.exports = ContentAssessor;
+require( "util" ).inherits( ContentAssessor, Assessor );
+
+ContentAssessor.prototype.calculateNegativePoints = function() {
+	var results = this.getValidResults();
+
+	var negativePoints = map( results, function( result ) {
+		var weight;
```
Comment (andizer) [inline js/contentAssessor.js]: You can combine this like you did on line 85

File: js/contentAssessor.js [hunk 4]
```diff
@@ -45,7 +50,64 @@ var ContentAssessor = function( i18n, options ) {
...
+		var rating = scoreToRating( result.getScore() );
+
+		// Convert the ratings to negative 'points'.
+		switch ( rating ) {
+			case "bad":
+				weight = 1;
+				break;
+
+			case "ok":
+				weight = 1 / 2;
```
Comment (andizer) [inline js/contentAssessor.js]: What's wrong with 0.5 ?

File: js/contentAssessor.js [hunk 5]
```diff
@@ -45,7 +50,64 @@ var ContentAssessor = function( i18n, options ) {
...
+
+ContentAssessor.prototype.calculateNegativePoints = function() {
+	var results = this.getValidResults();
+
+	var negativePoints = map( results, function( result ) {
+		var weight;
+		var rating = scoreToRating( result.getScore() );
+
+		// Convert the ratings to negative 'points'.
+		switch ( rating ) {
```
Comment (andizer) [inline js/contentAssessor.js]: These weight are the same for every rating. Why not declaring an array like: `{ "bad" : 1, "ok" : 0.5, "good" : 0 }` for determine the weight.

File: js/contentAssessor.js [hunk 6]
```diff
@@ -45,7 +50,64 @@ var ContentAssessor = function( i18n, options ) {
...
+		}
+
+		return weight;
+	} );
+
+	return sum( negativePoints );
+};
 
-require( "util" ).inherits( module.exports, Assessor );
+ContentAssessor.prototype.calculateOverallScore = function() {
```
Comment (andizer) [inline js/contentAssessor.js]: Missing documentation.

File: js/contentAssessor.js [hunk 7]
```diff
@@ -45,7 +50,64 @@ var ContentAssessor = function( i18n, options ) {
 	];
 };
 
-module.exports = ContentAssessor;
+require( "util" ).inherits( ContentAssessor, Assessor );
+
+ContentAssessor.prototype.calculateNegativePoints = function() {
```
Comment (andizer) [inline js/contentAssessor.js]: Missing documentation.

File: js/contentAssessor.js (hunk 5 above)
Comment (atimmer) [inline js/contentAssessor.js]: Because I want to have a default.

File: js/contentAssessor.js (hunk 4 above)
Comment (atimmer) [inline js/contentAssessor.js]: This makes it more explicit

Comment (andizer) [inline js/contentAssessor.js]: Okay 👍

**Comment by andizer** [comment]
CR 2 done 😄

**Comment by terw-dan** [comment]
Acceptance :+1:

**Pull request merged**